lib/route.coffee

```diff
@@ -159,11 +159,12 @@ class @Route
     auth = @api.config.auth.user.call(endpointContext)
 
     # Get the user from the database
-    if not auth?.user and auth?.userId and auth?.token
-      userSelector = {}
-      userSelector._id = auth.userId
-      userSelector[@api.config.auth.token] = auth.token
-      auth.user = Meteor.users.findOne userSelector
+    if not auth?.user
+      if auth?.userId and auth?.token
```

Comment:
That's fine. I updated it.
I am not fond of long if statements, since they suggest that we are doing too many things at once, and can also result in missed comprehension.
In any case, it works for me for now.